pytorch_lightning/trainer/connectors/training_trick_connector.py

```diff
@@ -32,7 +35,15 @@ def on_trainer_init(
         self.trainer.terminate_on_nan = terminate_on_nan
 
         # gradient clipping
+        regex = '^norm[1-9]([0-9]{0,45}$)'
+        if gradient_clip_algorithm != 'value' and re.match(regex, gradient_clip_algorithm) is None:
```

Comment:
Why should we use "norm[0-9]+" pattern instead of just "norm" for gradient_clip_algorithm?